CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 * Fixed error for partial result option if field contains box.NULL.
 
+### Added
+
+* `cut_rows` and `cut_objects` functions to cut off scan key and primary key values that were merged to the select/pairs partial result.
```

Comment:
Generally, not only select/pairs.

... merged to the result of operations with `fields` specified?